ldclient/__init__.py

```diff
@@ -340,7 +358,10 @@ def _headers(api_key):
     return {'Authorization': 'api_key ' + api_key, 'User-Agent': 'PythonClient/' + __version__, 'Content-Type': "application/json"}
 
 def _stream_headers(api_key):
-    return {'Authorization': 'api_key ' + api_key, 'User-Agent': 'PythonClient/' + __version__, 'Accept': "text/event-stream"}
+    return {'Authorization': 'api_key ' + api_key,
+            'User-Agent': 'PythonClient/' + __version__,
```

Comment:
It would be nice to set the `User-Agent` to `TwistedPythonClient` since the codebases are so different